src/index.js

```diff
@@ -0,0 +1,172 @@
+        temperature.style.color="#378aea"
+    }else if (temp <= 39 && temp >= 30){  
+        temperature.style.color="blue"        
+    }else {
+        temperature.style.color="#1100ab"
+    }           
+};
+
+function seasonChange(){
+    if (temp >= 80){
+        summer.style.display="flex"
+        spring.style.display="none"
+        fall.style.display="none"
+        winter.style.display="none"
+    }else if (temp >= 65){
+        summer.style.display="none"
+        spring.style.display="flex"
+        fall.style.display="none"
+        winter.style.display="none"
+    }else if (temp >= 45){
+        summer.style.display="none"
+        spring.style.display="none"
+        fall.style.display="flex"
+        winter.style.display="none"
+    }else if (temp <= 44){
+        summer.style.display="none"
+        spring.style.display="none"
+        fall.style.display="none"
+        winter.style.display="flex"
+    }
+};
+
+// Func for users city search and display name
+function displayInput() {
+    cityName.textContent = userSearch.value;
+}
+
+function resetCity() {
+    userSearch.value = 'Columbus';
+    cityName.textContent = 'Columbus';
+    getApi();
+}
+
+displayInput()
+seasonChange()
+
+
+// Function for getting value of sky/conditions dropdown
+function dropdownValue() {
+        console.log(skyDropDown.value);
+        console.log('1');
+    // if(skyDropDown.value === 'Select Conditions'){
+    //     sunnySky.style.display = 'none';
+    //     cloudySky.style.display = 'none';
+    //     rainySky.style.display = 'none';
+    //     snowSky.style.display = 'none';
+    //     defaultSky.style.display = 'block';
+    } if (skyDropDown.value === 'Sunny'){
+        Sunny.style.display = 'block'
+        Cloudy.style.display = 'none';
+        Rainy.style.display = 'none';
+        Snowy.style.display = 'none';
+        defaultSky.style.display = 'none';
+    } else if (skyDropDown.value === 'Rainy') {
+        Sunny.style.display = 'none'
+        Cloudy.style.display = 'none';
+        Rainy.style.display = 'block';
+        Snowy.style.display = 'none';
+        defaultSky.style.display = 'none';
+    } else if (skyDropDown.value === 'Cloudy') {
+        Sunny.style.display = 'none'
+        Cloudy.style.display = 'none';
+        Rainy.style.display = 'none';
+        Snowy.style.display = 'block';
+        defaultSky.style.display = 'none';
+    } else if (skyDropDown.value === 'Snowy') {
+        Sunny.style.display = 'none';
+        Cloudy.style.display = 'none';
+        Rainy.style.display = 'none';
+        Snowy.style.display = 'block';
+        defaultSky.style.display = 'none';
+    };
+
+dropdownValue()
+
+upArrow.addEventListener('click', increment);
+downArrow.addEventListener('click', decrement);
+resetCityBtn.addEventListener('click', resetCity);
+getWeatherBtn.addEventListener('click', getApi);
+userSearch.addEventListener('input', displayInput);
+skyDropDown.addEventListener('change', dropdownValue);
```

Comment:
Here the only suggestion is to move these inside a `registerHandlers` function that is made the callback to the`DOMContentLoaded`event handler.